engine/orchestration/src/com/cloud/vm/VirtualMachineManagerImpl.java

```diff
@@ -1090,8 +1091,8 @@ public void orchestrateStart(final String vmUuid, final Map<VirtualMachineProfil
                         break;
                     }
 
-                } catch (final OperationTimedoutException e) {
-                    s_logger.debug("Unable to send the start command to host " + dest.getHost());
+                } catch (OperationTimedoutException e) {
+                    s_logger.debug("Unable to send the start command to host " + dest.getHost()+" failed to start VM: "+vm.getUuid());
```

Comment:
@bvbharatk Can you please change it to 'error' so we can conclude the review and merge?